src/components/app/details/cIDetails/CIDetails.tsx

```diff
@@ -56,16 +58,20 @@ export default function CIDetails({ isJobView }: { isJobView?: boolean }) {
     useInterval(pollHistory, 30000)
 
     useEffect(() => {
-        if (!triggerHistoryResult) {
+        if (!triggerHistoryResult?.result?.ciWorkflows) {
             return
         }
-        if (triggerHistoryResult.result?.length !== pagination.size) {
+        if (triggerHistoryResult.result.ciWorkflows?.length !== pagination.size) {
             setHasMore(false)
         } else {
             setHasMore(true)
             setHasMoreLoading(true)
         }
-        const newTriggerHistory = (triggerHistoryResult.result || []).reduce((agg, curr) => {
+        const appReleaseTags = triggerHistoryResult.result?.appReleaseTagNames
+        const tagsEditable = triggerHistoryResult.result?.tagsEditable
+        setAreTagsEditable(tagsEditable)
```

Comment:
we can directly assign the values, no need to create variables

Reply:
done